ipapython/ipaldap.py

```diff
@@ -103,7 +103,7 @@ def realm_to_ldapi_uri(realm_name):
     return 'ldapi://' + ldapurl.ldapUrlEscape(socketname)
 
 
-def ldap_initialize(uri, cacertfile=None):
+def ldap_initialize(uri, cacertfile=None, ssf_min_threshold=56):
```

Comment:
Hi @tiran, 

Just asking, could we use ipalib/constants.py to contain this minimal threshold value?

Reply:
Done